cmd/utils.go

```diff
@@ -2322,3 +2322,39 @@ func RemoveIfExists(path string) error {
 	}
 	return nil
 }
+
+func generateJSON(log *LoggingConfig, indexYaml IndexYaml, indexFilePath string) error {
+	indexJSONStack := make([]IndexJSONStack, 0)
+	for _, stack := range indexYaml.Stacks {
+		for _, template := range stack.Templates {
+			stackJSON := IndexJSONStack{}
+			stackJSON.DisplayName = "Appsody " + stack.Name + " " + template.ID + " template"
+			stackJSON.Description = stack.Description
+			stackJSON.Language = stack.Language
+			stackJSON.ProjectType = "appsodyExtension"
+			stackJSON.ProjectStyle = "Appsody"
+			stackJSON.Location = template.URL
+
+			link := Link{}
+			link.Self = "/devfiles/" + stack.ID + "/devfile.yaml"
```

Comment:
As @CameronMcWilliam mentioned, could we use `filepath.Join`?

Reply:
Using `filepath.Join` on Windows would result in backslashes, which we don't want. See [here](https://github.com/kabanero-io/codewind-templates/blob/master/devfiles/index.json) for a valid Codewind index